docs/user/commercial/index.rst

```diff
@@ -1,10 +1,41 @@
-Read the Docs for Business
---------------------------
+About |com_brand|
+=================
 
-Read the Docs is our community solution for open source projects at `readthedocs.org`_
-and we offer Read the Docs for Business
-for building and hosting commercial documentation at `readthedocs.com`_.
-Features in this section are specific to Read the Docs for Business.
+.. this page is currently moving towards becoming "About Read the Docs for Business"
+.. rather than an index of features.
+
+We offer *|com_brand|*
+for building and hosting commercial documentation.
+
+Our commercial solutions are provided as a set of subscriptions that are paid and managed through an online interface.
+In order to get started quickly and easily, a `trial option <https://about.readthedocs.com/pricing/>`__ is provided free of charge.
+
+.. seealso::
+
+   `Read the Docs website: Features <https://about.readthedocs.com/about/>`__
+      A general presentation of our features is available on our website, and the
+      `pricing page <https://about.readthedocs.com/pricing/>`__ lists all details and features for each subscription level.
+
+   `Read the Docs website: Company presentation <https://about.readthedocs.com/company/>`__
+      Information about the company running Read the Docs, including our mission, team, and community.
```

Comment:
```suggestion
   `Read the Docs website: Features <https://about.readthedocs.com/about/>`__
      A high-level overview of platform features is available on our website, and the
      `pricing page <https://about.readthedocs.com/pricing/>`__ has a feature breakdown by subscription level.

   `Read the Docs website: Company <https://about.readthedocs.com/company/>`__
      Information about the company running Read the Docs, including our mission, team, and community.
```

I don't love the `Read the Docs website` prefix, but like you, couldn't come up with anything better. I want something other than `RTD` on it tho, like `Company website: Features` or something? But this is fine for now.

Reply:
"Company website: Company" isn't great but "Company website: About" works well.

I'm not sure that it's nice in all cases, since presenting "Read the Docs" as a company or a platform or a community are all aspects of the website.

If we could find a nice "universal" way to refer to our website, we could put it in the style guide. I unfortunately also don't have any good ideas right now :(

Reply:
> I'm not sure that it's nice in all cases, since presenting "Read the Docs" as a company or a platform or a community are all aspects of the website.

Yea, definitely part of the complexity. The same word being multiple things is always going to lead to confusion.